objects/url.json

```
@@ -1,9 +1,10 @@
 {
-  "observable": 23,
   "caption": "Uniform Resource Locator",
-  "description": "The Uniform Resource Locator(URL) object describes the characteristics of a URL. Defined in <a target='_blank' href='https://datatracker.ietf.org/doc/html/rfc1738'>RFC 1738</a> and by D3FEND <a target='_blank' href='https://d3fend.mitre.org/dao/artifact/d3f:URL/'>d3f:URL</a>.",
-  "extends": "object",
   "name": "url",
+  "observable": 23,
+  "description": "The Uniform Resource Locator(URL) object describes the characteristics of a URL.",
```

```suggestion
  "description": "The Uniform Resource Locator (URL) object describes the characteristics of a URL.",
```
Add missing space between "Locator" and "(URL)".